### PR TITLE
updating homepage card for assembly with link to assembly 22 group

### PR DESCRIPTION
**Pull request by gordoncc**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -66,11 +66,19 @@ export default function Home({
...
               </p>
-              <Link href="https://assembly.urbit.org" passHref>
-                <a className="button-lg max-w-fit bg-green-400 text-white">
-                  Learn More
-                </a>
-              </Link>
+
+              <div class="flex flex-wrap">
+                <Link href="https://assembly.urbit.org" passHref>
+                  <a className="button-lg max-w-fit bg-wall-600 text-white mr-3 mb-8">
```
Comment (matildepark) [inline pages/index.js]: Let's add `target="_blank"` to this. It sees urbit.org in the link so it isn't opening in a new tab, but it belongs outside the urbit.org family of sites IA.

File: pages/index.js [hunk 2]
```diff
@@ -66,11 +66,19 @@ export default function Home({
...
+                    Learn More
+                  </a>
+                </Link>
+                <Link href="/groups/~bitdeg/assembly22" passHref>
+                  <a className="button-lg max-w-fit bg-transparent text-wall-600 border-2 border-wall-600">
```
Comment (matildepark) [inline pages/index.js]: Maybe `hover:bg-wall-600 hover:text-transparent`? Looks weird as is imo but up to you.

**Review by matildepark — APPROVED** [review]
Fine otherwise

**Comment by gordoncc** [comment]
good catches

**Pull request merged**